go/src/meguca/server/router.go

```diff
@@ -248,15 +248,21 @@ func healthCheck(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("God's in His heaven, all's right with the world"))
 }
 
-// Get HookTube title from ID
-func getHookTubeTitle(w http.ResponseWriter, r *http.Request) {
-	s, err := goscraper.Scrape(
-		"https://hooktube.com/embed/"+extractParam(r, "id"), 3)
+// Get YouTube title and googlevideo URL from URL
+func getYouTubeData(w http.ResponseWriter, r *http.Request) {
+	info, err := ytdl.GetVideoInfo(("https://www.youtube.com/watch?v=" + extractParam(r, "id")))
 
 	if err != nil {
 		httpError(w, r, err)
 		return
 	}
 
-	w.Write([]byte(s.Preview.Title))
+	video, err := info.GetDownloadURL(info.Formats.Best("mp4")[0])
```

Comment:
Why not webm?

Reply:
The ytdl documentation is sparse in examples, and the documentation that is there is hard to understand the flow. Until I understand ytdl better, I want to just stick with mp4, before eventually making a switch for mp4/webm/whatever, since I have no way of knowing which format has the best resolution, and mp4 is the one that tends to.

Reply:
To put it into perspective, here's an image. I don't have a good way yet of determining which is best other than what's there, how do I compare the webm fetch with the mp4 fetch?
![image](https://user-images.githubusercontent.com/30435868/42834023-6f3a9460-89bb-11e8-8c47-acaf6550020d.png)
(it was just some random video on my feed)

Reply:
Why would you want the best?

Reply:
Because for youtube, the best is max 720p and then after that you have to combine sound and video streams, which I'm not going to write an entire custom player for.
HookTube had the same issue.

Reply:
Yes, but the embeds in meguca are size 360p. I don't see a reason for fuckhuge embeds inside posts.

Reply:
Because of the ability to make the video fullscreen.

Reply:
That will rape mobileposters savagely in the ass just for clicking expand.

Reply:
This wasn't a problem before with the HookTube commit, are you saying you want to remove the ability to do fullscreen, and have the embed default to 360p?

Reply:
I never realised Hooktube always played such huge videos. I want the embed to default to 360p. After that you can switch resolutions on fullscreen.

Reply:
And please use webm at all times. MP4 is a non-streamable container and should not be used on the web for that reason.

Reply:
Well, okay, thanks for being clear.
Do you know any way to switch resolutions on fullscreen by any chance?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Fullscreen_API and swicthing the src attibute should do it.